liquibase-core/src/main/java/liquibase/command/core/helpers/DbUrlConnectionCommandStep.java

```diff
@@ -0,0 +1,95 @@
+package liquibase.command.core.helpers;
+
+import liquibase.command.*;
+import liquibase.configuration.ConfigurationValueObfuscator;
+import liquibase.database.Database;
+import liquibase.exception.DatabaseException;
+import liquibase.util.StringUtil;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Internal command step to be used on CommandStep pipeline to manage the database connection.
+ */
+public class DbUrlConnectionCommandStep extends AbstractDatabaseConnectionCommandStep implements CleanUpCommandStep {
+
+    protected static final String[] COMMAND_NAME = {"dbUrlConnectionCommandStep"};
+
+    public static final CommandArgumentDefinition<Database> DATABASE_ARG;
+    public static final CommandArgumentDefinition<String> URL_ARG;
+    public static final CommandArgumentDefinition<String> DEFAULT_SCHEMA_NAME_ARG;
+    public static final CommandArgumentDefinition<String> DEFAULT_CATALOG_NAME_ARG;
+    public static final CommandArgumentDefinition<String> USERNAME_ARG;
+    public static final CommandArgumentDefinition<String> PASSWORD_ARG;
+    public static final CommandArgumentDefinition<String> DRIVER_ARG;
+    public static final CommandArgumentDefinition<String> DRIVER_PROPERTIES_FILE_ARG;
+
+    public static final CommandArgumentDefinition<Boolean> SKIP_DATABASE_STEP_ARG;
```

Comment:
What is `SKIP_DATABASE_STEP_ARG` used for?

Reply:
It's used to skip this step. I didn't come up with a good way yet to remove/skip a step that was added before on the pipeline, so I added this option to skip it here. I'll mark as beta/deprecated so people don't use it yet.